### PR TITLE
🍕 Scroll to top on element set change.

### LINKED ISSUE #55: Element set page scroll position doesn't reset when switching sets
To reproduce:
1. Select iron-elements
2. Scroll down
3. Select paper-elements

The paper-elements list should open at the top of the page, but it doesn't.

### PR DESCRIPTION
**Pull request by samccone**
Fixes #55

### CODE REVIEW

File: app/elements/pages/page-browse.html [hunk 1]
```diff
@@ -290,6 +291,9 @@ <h2 class="flex"><span>[[pageTitle]]</span> <span class="count">[[_elementCount]
       cartOpen: function(e) {
         this.fire('cart-open');
       },
+      scrollTop: function() {
```
Comment (frankiefu) [inline app/elements/pages/page-browse.html]: This overrides the `scrollTop` property.  It should be changed to something else like `scrollToTop`

Comment (samccone) [inline app/elements/pages/page-browse.html]: oh good call :+1: will fix

**Comment by samccone** [comment]
all set @frankiefu thanks for the great catch :clap: :cake:

**Comment by frankiefu** [comment]
Thanks for the quick fix.  Seems like there is a merge conflict.  Could you rebase it?

**Comment by samccone** [comment]
sure thing

**Comment by samccone** [comment]
all set :tanabata_tree:

**Comment by frankiefu** [comment]
LGTM :+1:

**Pull request merged**